content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -92,28 +62,19 @@ Since this release, the `c8y_LatestMeasurements` fragment will be ignored and no
 
 ##### Breaking change in all REST APIs
 
-As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of release 10.18+, wrong values for `pageSize` or `currentPage` result in a 422 (Unprocessable entity) HTTP status code instead of a 500 (Internal server error) HTTP status code.
-
-### Security changes
-
-#### Planned
-
-##### User administrator can no longer set password for other users
-
-As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), to improve security, as of a future releases, user administrators will no longer be able to explicitly set passwords for other users in the tenant.
-This change prevents that an attacker could have access to all users, in case the administrator account was compromised.
-Note that the administrator will still have the option to force the user to reset the password on the next login or disable the user.
-
+As announced earlier, see [release 10.17](/release-10-17-0/announcements-10-17-0), wrong values for `pageSize` or `currentPage` now result in a 422 (Unprocessable entity) HTTP status code instead of a 500 (Internal server error) HTTP status code.
 
 ### Microservice infrastructure changes
 
 #### Planned
 
 ##### Breaking change: Use of Linux cgroup v2
-As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), from release 2024, microservices must use a Linux cgroup v2 aware application runtime. When executing microservices which are not compatible with cgroup v2 on Cumulocity IoT in a version higher than 10.18 it might happen that the information provided by the application runtime concerning available CPU and memory is not correct. This might lead to incorrect memory and thread allocation in the microservice container process.
+
+As announced earlier, see [release 10.17](/release-10-17-0/announcements-10-17-0), in Cumulocity IoT versions higher than 10.18, microservices must use a Linux cgroup v2 aware application runtime. When executing microservices which are not compatible with cgroup v2 on Cumulocity IoT in these versions it might happen that the information provided by the application runtime concerning available CPU and memory is not correct. This might lead to incorrect memory and thread allocation in the microservice container process.
```

Comment:
```suggestion
As announced earlier, see [release 10.17](/release-10-17-0/announcements-10-17-0), in Cumulocity IoT versions after 10.18, microservices must use a Linux cgroup v2 aware application runtime. When executing microservices which are not compatible with cgroup v2 on Cumulocity IoT in these versions it might happen that the information provided by the application runtime concerning available CPU and memory is not correct. This might lead to incorrect memory and thread allocation in the microservice container process.
```